### ISSUE TITLE
Non primitive default values that get serialized as JSON do not match the property name casing of the generated files

### ISSUE THREAD

**Issue by cs-NET** [jburzynski/TypeGen#146]
Current Output:
```
tracker: Tracker = {"Index":null,"Name":null};
```

Expected Output:
```
tracker: Tracker = {"index":null,"name":null};
```

Looking at TsContentGenerator.cs, a possible solution would be passing in serializing settings when calling JsonConvert.SerializeObject(valueObj) that match the property name converter casing.

**Comment by cs-NET** [comment]
Created PR #149 to address this issue

**Comment by cs-NET** [comment]
Noticed duplicate of issue #135

**Comment by jburzynski** [comment]
fixed in #149
